### PR TITLE
[WIP] Add ResNet101 based Faster R-CNN

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

**Comment by yuyu2172** [comment]
#### Memo

Train:  VOC07trainval    Val:  VOC07test
mAP 0.7332430230005217

**Comment by yuyu2172** [comment]
I was mistakenly using max-pooling instead of avg-pooling after res5.

**Comment by yuyu2172** [comment]
wip https://github.com/yuyu2172/chainercv/tree/faster-rcnn-resnet1

File: docs/source/license.rst [hunk 1]
```diff
@@ -27,6 +27,11 @@ See the following resources for the terms of use of a model with weights pretrai
     * - FasterRCNNVGG16 (:obj:`voc07`/:obj:`voc0712`)
       - * `VGG-16 (trained on ImageNet) <http://www.robots.ox.ac.uk/%7Evgg/research/very_deep/>`_
         * `PASCAL VOC <http://host.robots.ox.ac.uk/pascal/VOC/>`_
+    * - FasterRCNNResNet101 (:obj:`imagenet`)
+      - * `ResNet50/101/152 (trained on ImageNet) <https://github.com/KaimingHe/deep-residual-networks#models>`_
```
Comment (Hakuyume) [inline docs/source/license.rst]: `ResNet101 (trained on ImageNet)`? Does it use ResNet50/152?

**Pull request closed without merging**